lib/cli/src/cli.rs

```diff
@@ -151,6 +153,10 @@ enum WasmerCLIOptions {
     #[clap(name = "binfmt")]
     Binfmt(Binfmt),
 
+    /// Shows the current logged in user for the current active registry
+    #[clap(name = "whoami")]
```

Comment:
fyi - if you don't use the `#[clap(name = "...")]` attribute, `clap` will default to using the lowercased version of the enum variant for a subcommand's name (i.e. `whoami`) so we don't need to add the attribute.

Reply:
Yeah I just did it to stay consistent with the current implementation, I know that it does do that.

Should I remove it?

Reply:
okay I removed it